src/pages/ArticlesPage.jsx

```diff
@@ -5,17 +5,49 @@ import AllArticles from "../components/AllArticles";
 
 export default function ArticlesPage() {
   const [existingArticles, setArticles] = useState([]);
+  const [order, setOrder] = useState("DESC");
+  const [sort, setSort] = useState("created_at");
   const [loading, isLoading] = useState(true);
   const { topic } = useParams();
 
   useEffect(() => {
     isLoading(true);
-    getArticles(topic).then(({ articles }) => {
+    getArticles(topic, sort, order).then(({ articles }) => {
       setArticles(articles);
       isLoading(false);
     });
-  }, [topic]);
+  }, [topic, sort, order]);
+
+  const handleOrder = (e) => {
+    setOrder(e.target.value);
+  };
+  const handleSort = (e) => {
+    setSort(e.target.value);
+  };
 
   if (loading) return <h2>loading...</h2>;
-  return <AllArticles existingArticles={existingArticles} />;
+  return (
```

Comment:
This is working, but you may have noticed the drop down menu defaults back to Date after it's been submitted. This is because it should be a controlled component, where the value of the select bar is saved as its own state.